templates/assets/css/vlabs-style.css

```diff
@@ -156,6 +156,7 @@
   font-family: "Raleway", sans-serif;
   color: #000000;
   text-align: justify;
+  hyphens: auto;
```

Comment:
Is this change going in with this commit?

Reply:
I didn't notice that, I'll remove it.